pkg/knative/integration_test.go

```diff
@@ -127,10 +127,11 @@ func TestIntegration(t *testing.T) {
 		//     * environment variables starting which name starts with FUNC_TEST,
 		//     * files under /etc/cm and /etc/sc.
 		//   * application also prints the same info to stderr on startup
-		Image:       "quay.io/mvasek/func-test-service",
-		ImageDigest: "sha256:2eca4de00d7569c8791634bdbb0c4d5ec8fb061b001549314591e839dabd5269",
-		Created:     now,
+		Created: now,
 		Deploy: fn.DeploySpec{
+			// TODO: gauron99 - is it okay to have this explicitly set to deploy.image already?
+			// With this I skip the logic of setting the .Deploy.Image field but it should be fine for this test
+			Image:     "quay.io/mvasek/func-test-service@sha256:2eca4de00d7569c8791634bdbb0c4d5ec8fb061b001549314591e839dabd5269",
```

Comment:
opening this TODO for comments

Reply:
I would prefer this test used the complete path, and the `fn` API, rather than hard-coding the function struct, but it appears to perhaps be a test performance enhancement.

Perhaps leave it as-is (as you have done here) for this PR and we can confirm with Matej on our next WG call.